projects/plugins/jetpack/modules/search/instant-search/components/search-app.jsx

```diff
@@ -144,11 +155,15 @@ class SearchApp extends Component {
 		document.body.style.overflowY = null;
 	}
 
-	scrollOverlayToTop() {
-		const overlay = document.querySelector( `.${ OVERLAY_CLASS_NAME }` );
+	scrollOverlayToTop( event ) {
+		if ( event?.propertyName !== 'transform' ) {
+			return;
+		}
+		const overlay = event.target;
 		// NOTE: IE11 doesn't support scrollTo. Manually set overlay element's scrollTop.
 		if ( overlay.scrollTo ) {
-			overlay.scrollTo( 0, 0, { smooth: true } );
+			// @see https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo
+			overlay.scrollTo( 0, 0 );
```

Comment:
Is `{ smooth: true }` no longer required? Can we use the options syntax instead, like:

```
element.scrollTo({
  top: 100,
  left: 100,
  behavior: 'smooth'
});
```

Reply:
Safari does not support scrollTo options, so I  don't think we'd use options - and I don't see any difference whether I add smooth or not.  https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo#browser_compatibility

Reply:
hmmm I tested on latest Safari, it actually supports scrollTo options.

Reply:
I wonder how it'd fare on older versions of Safari 🤔 Could be worth a go on Browserstack?

Also happy to omit it if it makes no visual difference :)

Reply:
I reckon we use scrollTo without options, which has better compatibility and has no virtual difference via behaviour `smooth` parameter. 😄